codec-http2/src/main/java/io/netty/handler/codec/http2/HpackDecoder.java

```diff
@@ -408,6 +411,20 @@ private static HeaderType validate(int streamId, CharSequence name,
         return HeaderType.REGULAR_HEADER;
     }
 
+    private static boolean contains(Http2Headers headers, CharSequence name) {
+        try {
+            return headers.contains(name);
+        } catch (Throwable e) {
```

Comment:
uff… while this will “fix it” it will also cause a lot of overhead as we will produce a lot of exceptions. Maybe we should just use get when a system property is set ?